clouddq/classes/dq_entity.py

```diff
@@ -362,6 +362,31 @@ def from_dataplex_entity(
                 f"is unsupported for entity:\n {dataplex_entity.to_dict()}"
             )
 
+    @classmethod
+    def from_bq_table_configs(
```

Comment:
this should take the bigquery entity_uri directly.

Reply:
updated